app/controllers/response_sets_controller.rb

```diff
@@ -46,6 +47,22 @@ def create
     end
   end
 
+  def transfer_table
+    @response_set.coded = @parent_set.coded
+    @response_set.oid = @parent_set.oid
+    @response_set.description = @parent_set.description
+    @response_set.name = @parent_set.name
+    @parent_set.responses.each do |r|
```

Comment:
yes, but instead of copying the responses they could simply be obtained from the parent.  So the total responses for a response_set would be the union of the parent's and the child's